File: code/unclassified/src/biggest_of_n_numbers/biggest_of_n_numbers.cs
namespace BiggestOfNumbers
{
    public class BiggestOfNumbers
    {
        public void main()
        {
            int n , max , tmp;
            Console.WriteLine("Enter numbers of elements : ");
            n = Console.ReadLine();
            Console.WriteLine("Enter numbers : ");
            tmp = Console.ReadLine();
            max = tmp;
            for(int i=0; i<n-1; i++)
            {
                tmp = Console.ReadLine();
                if (max<tmp)
                {
                    max = tmp;
                }
            }
            Console.WriteLine("Maximum is {0}",max);
        }
    }
}

File: code/unclassified/src/biggest_of_n_numbers/Biggest_of_n_numbers.java
import java.util.Scanner;

public class Biggest_of_n_numbers
  {
  public static void main(String argn[]){
    Scanner data = new Scanner(System.in);
    int n, i, temp, max;
    System.out.print("Enter numbers of elements");
    n = data.nextInt();
    System.out.println("Enter " +n+ " numbers");
    max = data.nextInt();
    for (i = 1; i < n; ++i) 
{ 
    temp = data.nextInt();
    if(temp < max)
    continue;
    else
    max = temp;
}
   System.out.println("Maximum is "+max);
}
}